common/src/main/java/com/radixdlt/sbor/coding/Decoder.java

```diff
@@ -242,6 +242,14 @@ public String readString() {
   public byte[] readBytes(int length) {
     var bytes = new byte[length];
 
+    // Early return if the length is 0.
+    // This is needed because, contrary to JavaDoc,
```

Comment:
This is **_FALSE ACCUSATION_**, they DO NOT "lie in the javadocs"...

... they do something MUCH WORSE: they redefine the contract of the overridden method :D

![image](https://github.com/radixdlt/babylon-node/assets/119413677/f5bf6825-d836-4bf0-8690-86c27621ae90)

good catch 👍

Reply:
omg :man_facepalming: I updated the comment. We don't want no false accusations here